test/jdk/java/nio/file/Path/ToRealPath.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 8295753
+ * @summary Verify correct operation of Path.toRealPath
+ * @library .. /test/lib
+ * @build ToRealPath jdk.test.lib.Platform
+ * @run junit ToRealPath
+ */
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import jdk.test.lib.Platform;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIf;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import static java.nio.file.LinkOption.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ToRealPath {
+    static final boolean SUPPORTS_LINKS;
+    static final Path DIR;
+    static final Path SUBDIR;
+    static final Path FILE;
+    static final Path LINK;
+
+    static {
+        try {
+            DIR  = TestUtil.createTemporaryDirectory();
```

Comment:
```suggestion
            DIR = TestUtil.createTemporaryDirectory();
```

Reply:
Fixed in 34511d8814856f6eedbf742ed959533f3592bebb, thanks.